packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -206,17 +210,25 @@ void _writeIOSPluginRegistry(String directory, List<Plugin> plugins) {
 
 }
 
+class InjectPluginsResult{
+  // True if any flutter plugin exists, otherwise false.
+  bool anyPlugin;
+  // True if plugins have changed.
+  bool hasChanged;
+  InjectPluginsResult({this.anyPlugin = false,this.hasChanged=false});
```

Comment:
nit: let the constructor be on top, use : for defaults, like `this.anyPlugin: false`. Space after `,`